pyomo/core/tests/unit/test_set.py

```diff
@@ -4347,7 +4347,10 @@ def _validate(model, i, j):
 
         # Note: one of these indices will trigger the exception in the
         # validot when it is called for the index.
-        m.J = Set([(0, 0), (2, 2)], validate=_validate)
+        def _validate_J(model, i, j, index):
```

Comment:
I envision that future developers looking at this code will wonder, "What is `J`?"

Reply:
In my mind, it is just an indexed set that we validate through this new rule.